pkg/services/storage/catalog.go

```diff
@@ -180,5 +180,41 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 				},
 			}),
 		),
+		service.NewService(
+			service.ServiceProperties{
+				ID:              "fb6ce656-c16d-4b48-aff9-286714298af8",
+				Name:            serviceBlobContainer,
+				Description:     "A blob container inside an existing blob storage account",
+				ParentServiceID: "1a5b4582-29a3-48c5-9cac-511fd8c52756",
+				Metadata: service.ServiceMetadata{
+					DisplayName: "Azure Storage",
```

Comment:
If I remember correctly, PCF Apps Manager (maybe more tools) shows this name in the UI. Users would get several different services with the same name.

Reply:
Please also take a look at the field in other services in this module. That might be ignored in the last PR :).